felix/fv/health_test.go

```diff
@@ -223,6 +222,7 @@ var _ = Describe("_HEALTH_ _BPF-SAFE_ health tests", func() {
 				"-e", "FELIX_PROMETHEUSMETRICSENABLED=true",
 				"-e", "FELIX_USAGEREPORTINGENABLED=false",
 				"-e", "FELIX_DEBUGMEMORYPROFILEPATH=\"heap-<timestamp>\"",
+				"-e", "FELIX_DataplaneWatchdogTimeout=20",
```

Comment:
It would be nice to add a test for when the timeout is set to a larger value.  (Or is the new default of 90s.)